python/private/bzlmod/pip_repository.bzl

```diff
@@ -27,10 +27,11 @@ exports_files(["requirements.bzl"])
 def _pip_repository_impl(rctx):
     bzl_packages = rctx.attr.whl_map.keys()
     aliases = render_pkg_aliases(
-        repo_name = rctx.attr.repo_name,
-        rules_python = rctx.attr._template.workspace_name,
+        aliases = {
+            key: [whl_alias(**v) for v in json.decode(values)]
```

Comment:
Out of scope for this PR, but I wonder if passing a file might work better than passing it as strings in regular attributes. The pip hub basically needs to be given **all** the possible backing repos and targets, right? Which could be rather large. Putting that in a file seems like it'd scale better than the string attributes.

Oh hm...and that sort of sounds like the sort of intermediate file Phil's thing has been circling around...

Reply:
Interesting, I like the file idea, we could have a `label-keyed` dict here. Once I merge this, I can experiment with this idea.